src/microbe_stage/ColonyCompoundBag.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+public class ColonyCompoundBag : CompoundBag
```

Comment:
What's the reason why this class exists? I can kinda see why this is an useful interface to read the entire colony data through. Refer to my previous comment.
This class should be documented.